lib/export/export_recurring_donations.rb

```diff
@@ -2,7 +2,7 @@
 
 module ExportRecurringDonations
 
-  def self.initiate_export(npo_id, params, user_id)
+  def self.initiate_export(npo_id, params, user_id, requested_by_user = true)
```

Comment:
I'm confused, why does "requested_by_user" matter here?

Reply:
> I don't know the difference between the two. Could you explain?

I don't know why I can't reply that comment, but that's the difference.